### DISCUSSION TITLE
Possible to implement detachable tabs?

### DISCUSSION THREAD

**Discussion by shi-yan** [rust-windowing/winit#2278] [Q&A]
I want to implement an app with detachable tabs, (similar to what chrome and sublime text have)

I wonder if this is possible with the current API?

Thanks.

**Comment by maroider** [comment]
While I haven't specifically looked into this myself, I don't believe it's possible with the current cross-platform API.

**Reply by shi-yan** [reply]
I know this is doable with SDL2.

I think one problem is that this function

https://docs.rs/winit/latest/winit/window/struct.Window.html#method.drag_window

doesn't allow canceling. It will only stop when the mouse is released.
